config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      get 'messages/index'
+    end
+    get 'v1/message'
+  end
+  get 'root/index'
```

Comment:
- [ ] Also, please remove this root route because it is already declared at the bottom correctly. 🙏